Makefile

```diff
@@ -56,41 +77,54 @@ docs/lvmd-protocol.md: lvmd/proto/lvmd.proto
 PROTOBUF_GEN = csi/csi.pb.go csi/csi_grpc.pb.go \
 	lvmd/proto/lvmd.pb.go lvmd/proto/lvmd_grpc.pb.go docs/lvmd-protocol.md
 
-.PHONY: test
-test:
-	test -z "$$(gofmt -s -l . | grep -v '^vendor' | tee /dev/stderr)"
-	$(STATICCHECK) ./...
-	test -z "$$($(NILERR) ./... 2>&1 | tee /dev/stderr)"
-	$(INEFFASSIGN) .
-	go install ./...
-	go test -race -v ./...
-	go vet ./...
-	test -z "$$(go vet ./... | grep -v '^vendor' | tee /dev/stderr)"
-
-# Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
-manifests:
+manifests: ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) \
 		crd:crdVersions=v1 \
 		rbac:roleName=topolvm-controller \
 		webhook \
-		paths="./api/...;./controllers;./hook;./driver/k8s" \
+		paths="./api/...;./controllers;./hook;./driver/k8s;./pkg/..." \
 		output:crd:artifacts:config=config/crd/bases
 	rm -f deploy/manifests/base/crd.yaml
 	cp config/crd/bases/topolvm.cybozu.com_logicalvolumes.yaml deploy/manifests/base/crd.yaml
 
 .PHONY: generate
-generate: $(PROTOBUF_GEN)
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./api/..."
+generate: $(PROTOBUF_GEN) ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+	$(CONTROLLER_GEN) object:headerFile="./hack/boilerplate.go.txt" paths="./api/..."
 
 .PHONY: check-uncommitted
-check-uncommitted:
+check-uncommitted: ## Check if latest generated artifacts are committed.
 	$(MAKE) manifests
 	$(MAKE) generate
 	git diff --exit-code --name-only
 
+.PHONY: lint
+lint: ## Run lint
+	test -z "$$(gofmt -s -l . | grep -v '^vendor' | tee /dev/stderr)"
+	$(STATICCHECK) ./...
+	test -z "$$($(NILERR) ./... 2>&1 | tee /dev/stderr)"
+	$(INEFFASSIGN) .
+	go vet ./...
+	test -z "$$(go vet ./... | grep -v '^vendor' | tee /dev/stderr)"
+
+.PHONY: test
+test: lint ## Run lint and unit tests.
+	go install ./...
+
+	mkdir -p $(ENVTEST_ASSETS_DIR)
+	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v$(CONTROLLER_RUNTIME_VERSION)/hack/setup-envtest.sh
```

Comment:
It seems that `setup-envtest.sh` downloads 1.19.2 env test tools by default. Is it OK not to use the minimum supported version of the tool?
https://github.com/kubernetes-sigs/controller-runtime/blob/v0.8.2/hack/setup-envtest.sh#L65

Reply:
Sorry, I was wrong. It's not necessary.
[coil](https://github.com/cybozu-go/coil/blob/main/v2/Makefile#L164) uses the minimum supported version of the tool because its tests didn't work with the bundled tools.  It's not the case for kubebuilder v3.